### PR TITLE
feat: React landing page.

### PR DESCRIPTION
**Pull request by aidanCQ**

### CODE REVIEW

**Comment by irfankhan10** [comment]
Landing page after modifying / adding typescript components + adding product content:

image

**Review by dimkart — CHANGES_REQUESTED** [review]
Still inquanto stuff in this PR.

File: landing/src/app/layout.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+const inter = Inter({ subsets: ['latin'], variable: '--font-inter' })
+
+const jetBrains = Roboto_Mono({
+  subsets: ['latin'],
+  variable: '--font-jetbrains-mono',
+})
+
+export const metadata: Metadata = {
+  title: '\u03BBambeq',
+  description: 'Quantum Chemistry on Quantum Computers',
```
Comment (dimkart) [inline landing/src/app/layout.tsx:16]: Natural Language Processing on Quantum Computers

**Comment by dimkart** [comment]
Delete:

[landing/public/inquanto_molecule.avif](https://github.com/CQCL/lambeq-docs/pull/15#diff-879b2f8d3c0e6baffe38980b35b24e42ef3389da7fe935d536e3929902400dd2)

File: landing/src/app/page.tsx [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+
+import Image from 'next/image'
+import Link from 'next/link'
+import { LambeqLogo } from './LambeqLogo';
+
+const navConfig = {
+  navTextLinks: [
+    {
+      title: 'Getting Started',
+      href: '/intro.html',
```
Comment (dimkart) [inline landing/src/app/page.tsx:13]: I guess all these need to be `docs/intro.html`, `docs/pipeline.html' etc? Locally none of these links work.

File: landing/src/app/page.tsx [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+    "title": "Getting Started",
+    "subtitle": "Learn the basics of \u03BBambeq and compositional models of meaning.",
+    icon: "",
+    "image_link": "",
+    "image_description": "",
+    "links": [
+        {
+            "title": "\u03BBambeq pipeline",
+            "description": "Understand how a sentence is converted into a quantum circuit.",
+            "link": "/pipeline.html"
```
Comment (dimkart) [inline landing/src/app/page.tsx:81]: These links as well.

File: landing/src/app/page.tsx (hunk 2 above)
Comment (aidanCQ) [reply landing/src/app/page.tsx:13]: works for me locally

File: landing/src/app/page.tsx (hunk 3 above)
Comment (aidanCQ) [reply landing/src/app/page.tsx:81]: Ah yes, these shouldn't be absolute, don't know why they were

**Comment by aidanCQ** [comment]
> @aidanCQ @irfankhan10 from my understanding, the `BASE_PATH="/lambeq"` variable is needed when building for the version that will be deployed (e.g. in the Github workflow) since the lambeq docs will sit in https://docs.quantinuum.com/lambeq. Is this right?
> 
> For testing with a local server, this environment variable can be empty so we can directly access with localhost:8000. This prevents the need to put everything inside a `build/lambeq` directory.

Correct, yes.

**Comment by aidanCQ** [comment]
> @aidanCQ @irfankhan10 One minor thing I noticed with the new navbar (the bar on the top): if I click on a link that goes to a specific part of the API page and triggers an autoscroll, the target location/content is hidden by the navbar. Is that easily fixable?

This would require further investigation tbh.

**Review by dimkart — APPROVED** [review]

**Comment by dimkart** [comment]
@neiljdo @aidanCQ I think this can be merged now, but please feel free to have a final look if you want. We'll need a docs release to trigger pulling from the unified docs repo after.

**Comment by aidanCQ** [comment]
> @neiljdo @aidanCQ I think this can be merged now, but please feel free to have a final look if you want. We'll need a docs release to make it public and trigger pulling from the unified docs repo after.

I'm happy to merge. I don't have permissions though.

**Comment by neiljdo** [comment]
@aidanCQ @dimkart I can take a final look today. If everything works as expected, I'll do the merge and new release, shortly.

**Review by neiljdo — APPROVED** [review]
Checked locally and everything looks/works as expected.

**Pull request merged**